tests/unit/test-cmd/test.run.js

```diff
@@ -187,6 +188,13 @@ describe('run', () => {
     assert.equal(reloadStrategy.called, false);
   });
 
+  it('Reload Delay', async () => {
```

Comment:
A code base with a consistent coding style is easier to work with. So when you introduce new code to an existing code base, try to follow the existing code style.

Although we have no strict requirements on the format of the test description, all other tests appear to start with a lowercase word, and are a bit more descriptive than just "Reload Delay".